Lib/unittest/case.py

```diff
@@ -84,6 +84,20 @@ def testPartExecutor(self, test_case, isTest=False):
 def _id(obj):
     return obj
 
+
+_module_cleanups = []
```

Comment:
Very minor nit: blank line between this and the `def`?

Reply:
I'll remove that :)

Reply:
s/remove/add